New version: IterativeRefinement v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: IterativeRefinement
- Repository: https://github.com/RalphAS/IterativeRefinement.jl
- Created by: @RalphAS
- Version: v1.0.0
- Commit: 3d94d779b48a186656cd34eff1565a9878cd8398
- Reviewed by: @RalphAS
- Reference: https://github.com/RalphAS/IterativeRefinement.jl/commit/3d94d779b48a186656cd34eff1565a9878cd8398#commitcomment-171923700
- Description: Multi-precision iterative refinement for linear matrix systems
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Stabilized API

No breaking changes have been made in a long time, so bumping to a version number suitable for stable code.

## New feature
Accommodate partial Schur decompositions for eigenvalue cluster refinement
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d09b57c63918b80648d4b281452e83962f518a987181286cd811a403bde2fbac6a2fd583e35bf5aacd09e69edb6e94ba604331ca48fd8b98c85fa6bb2f921b4676822bc69f789852fd1c6c5bf23f0db6c508a801f332bf8755a984c998b7dca69b320c1214d7fbbccddde2bfe13c0cb15290cbb36f24200bd81010a80cf3f86bc3854046fa0cb7bcb72565e7554b63635c36f9e7d62e2957b7379dcfb87859c9ee6de9a87734a6595938b375d9639895 -->

RalphAS commented:
@JuliaRegistrator register

Release notes:

## Stabilized API

No breaking changes have been made in a long time, so it is appropriate to use a version number for stable code.

## New feature
Accommodate partial Schur decompositions for eigenvalue cluster refinement

JuliaRegistrator committed:
New version: IterativeRefinement v1.0.0

UUID: bcb6b82e-7479-5933-8c8e-2e0b909ee6a3
Repo: https://github.com/RalphAS/IterativeRefinement.jl.git
Tree: caee17872b6a108c16ea4802382d7ef83c4750c7

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
